Config/CommonConfiguration.swift

```diff
@@ -66,8 +70,13 @@ class CommonConfiguration: NSObject, Configurable {
         // Disable identicon use
         sdkOptions.disableIdenticonUseForUserAvatar = true
         
+        // Set up user agent
+        if let userAgent = makeASCIIUserAgent() {
+            sdkOptions.httpAdditionalHeaders = ["User-Agent": userAgent]
+        }
+
         // Pass httpAdditionalHeaders to the SDK
-        sdkOptions.httpAdditionalHeaders = BuildSettings.httpAdditionalHeaders
+        sdkOptions.httpAdditionalHeaders = (sdkOptions.httpAdditionalHeaders ?? [:]).merging(BuildSettings.httpAdditionalHeaders, uniquingKeysWith: { _, value in value })
```

Comment:
We can make this more readable like:
```
var httpAdditionalHeaders = BuildSettings.httpAdditionalHeaders
// Set up user agent
if let userAgent = makeASCIIUserAgent() {
    httpAdditionalHeaders["User-Agent"] = userAgent
}
// Pass httpAdditionalHeaders to the SDK
sdkOptions.httpAdditionalHeaders = httpAdditionalHeaders
```

Reply:
Do we not have to do it the other way around so that forks can use `BuildSettings.httpAdditionalHeaders` to override the user agent?

Reply:
Yeah you right, i haven't considered that. Let's keep it as you've done.